### PR TITLE
Privacy Sandbox landing page, structure only

### PR DESCRIPTION
**Pull request by heyawhite**
Changes proposed in this pull request:

- Re-organizing the Privacy Sandbox doc pages into groups

Not covered in this pull request:

- Updates to the API descriptions to ensure brevity and clarity. 

Preview: https://deploy-preview-1843--developer-chrome-com.netlify.app/docs/privacy-sandbox/

### CODE REVIEW

File: site/_data/docs/privacy-sandbox/toc.yml [hunk 1]
```diff
@@ -1,11 +1,39 @@
...
+        - url: /docs/privacy-sandbox/attribution-reporting
+          title: i18n.docs.privacy-sandbox.attribution-reporting-overview
+          description: i18n.docs.privacy-sandbox.attribution-reporting_description
+        - url: /docs/privacy-sandbox/attribution-reporting-event-guide
+          title: i18n.docs.privacy-sandbox.attribution-reporting-events
+        - url: /docs/privacy-sandbox/attribution-reporting-data-clearing
+          title: i18n.docs.privacy-sandbox.attribution-reporting-faqs
+    - url: /docs/privacy-sandbox/floc
+    - url: /docs/privacy-sandbox/fledge
+- title: i18n.docs.privacy-sandbox.discuss
```
Comment (rowan-m) [inline site/_data/docs/privacy-sandbox/toc.yml:33]: Drop this section, I think it effectively repeats the blog link available at the top. RSS subscription is available from that page as well.

File: site/_data/docs/privacy-sandbox/toc.yml [hunk 2]
```diff
@@ -1,11 +1,39 @@
...
-- url: /docs/privacy-sandbox/user-agent
-- url: /docs/privacy-sandbox/floc
-- url: /docs/privacy-sandbox/fledge
+# External URLs need an explicit title and description, otherwise
+# the toc won't know what to display.
+- title: i18n.docs.privacy-sandbox.overview
+  sections:
+    - url: /docs/privacy-sandbox/overview
+      title: i18n.docs.privacy-sandbox.overview-article
+    - url: /docs/privacy-sandbox/cds21-update
```
Comment (rowan-m) [inline site/_data/docs/privacy-sandbox/toc.yml:7]: CDS update should be the last item in the intro section, as we should aim to break this up and create the more general process guidance, then the CDS update remains for historical reference but won't be a top-level item.

**Review by rowan-m — CHANGES_REQUESTED** [review]
First pass!

File: site/_data/docs/privacy-sandbox/toc.yml [hunk 3]
```diff
@@ -1,11 +1,39 @@
...
+          title: i18n.docs.privacy-sandbox.attribution-reporting-events
+        - url: /docs/privacy-sandbox/attribution-reporting-data-clearing
+          title: i18n.docs.privacy-sandbox.attribution-reporting-faqs
+    - url: /docs/privacy-sandbox/floc
+    - url: /docs/privacy-sandbox/fledge
+- title: i18n.docs.privacy-sandbox.discuss
+  sections:
+    - url: https://developer.chrome.com/feeds/privacy.xml
+      title: i18n.docs.privacy-sandbox.subscribe
+    - url: https://github.com/WICG/turtledove/issues
```
[the remark this answers is not in the document]
Comment (heyawhite) [reply site/_data/docs/privacy-sandbox/toc.yml:37]: Should have been https://github.com/GoogleChromeLabs/privacy-sandbox-dev-support and will be updated

File: site/_data/docs/privacy-sandbox/toc.yml (hunk 2 above)
Comment (heyawhite) [reply site/_data/docs/privacy-sandbox/toc.yml:7]: I imagine it transforming into "Collaboration" and dropping the CDS-specific context. But that said, I can move it for now as it remains CDS-specific.

File: site/_data/docs/privacy-sandbox/toc.yml [hunk 4]
```diff
@@ -1,11 +1,39 @@
...
+    - url: /tags/privacy/
+      title: i18n.docs.privacy-sandbox.blog
+    - url: https://privacysandbox.com/timeline/
+      title: i18n.docs.privacy-sandbox.timeline
+- title: i18n.docs.privacy-sandbox.start
+  sections:
+    - url: /docs/privacy-sandbox/glossary
+    - url: /docs/privacy-sandbox/status
+    - url: /docs/privacy-sandbox/faq
+- title: i18n.docs.privacy-sandbox.apis
```
[the remark this answers is not in the document]
Comment (heyawhite) [reply site/_data/docs/privacy-sandbox/toc.yml:17]: 👍🏻  I'm on it

[the remark this answers is not in the document]
Comment (heyawhite) [reply site/_data/docs/privacy-sandbox/toc.yml]: Done -- waiting on the new build.

[the remark this answers is not in the document]
Comment (heyawhite) [reply site/_data/docs/privacy-sandbox/toc.yml]: There's a small problem. "Strengthen cross-site privacy boundaries" is so long, it overlaps

Strengthen cross-site privacy boundaries section

[the remark this answers is not in the document]
Comment (heyawhite) [reply site/_data/docs/privacy-sandbox/toc.yml]: Shortening to "Strengthen privacy boundaries"

File: site/_data/docs/privacy-sandbox/toc.yml (hunk 1 above)
Comment (rowan-m) [reply site/_data/docs/privacy-sandbox/toc.yml:43]: Sorry, to clarify: I don't think we should have the support repo linked at this top-level. It should be in the individual articles.

Comment (rowan-m) [reply site/_data/docs/privacy-sandbox/toc.yml:43]: And so the whole "Discuss" section should just drop.

Comment (heyawhite) [reply site/_data/docs/privacy-sandbox/toc.yml:43]: Got it!

File: site/_data/docs/privacy-sandbox/toc.yml (hunk 4 above)
[the remark this answers is not in the document]
Comment (rowan-m) [reply site/_data/docs/privacy-sandbox/toc.yml]: LGTM. Longer-term we can, presumably, have some intro prose for each section that can clarify. Though, maybe we could just raise this as a feature improvement for the site. Like, that text could wrap.

[the remark this answers is not in the document]
Comment (heyawhite) [reply site/_data/docs/privacy-sandbox/toc.yml]: 1000% I'll open that issue

**Comment by heyawhite** [comment]
@rowan-m all pass 1 feedback has been addressed.

FYI for @rachelandrew, this is the structure-only PR (i.e. no updates to the descriptions of the proposals)

File: site/_data/i18n/docs/privacy-sandbox.yml [hunk 5]
```diff
@@ -10,7 +10,51 @@ guiding-principles:
...
   en: 'What is the Privacy Sandbox?'
+overview-article:
+  en: 'Introduction'
 status:
   en: 'Is it ready yet?'
 trust-tokens:
   en: 'Trust Tokens'
+start:
+  en: 'Get started'
+discuss:
```
Comment (rowan-m) [inline site/_data/i18n/docs/privacy-sandbox.yml:21]: Unused.

File: site/_data/i18n/docs/privacy-sandbox.yml [hunk 6]
```diff
@@ -10,7 +10,51 @@ guiding-principles:
...
+  en: 'Discussion and updates'
+issues:
```
Comment (rowan-m) [inline site/_data/i18n/docs/privacy-sandbox.yml:23]: Unused.

File: site/_data/i18n/docs/privacy-sandbox.yml [hunk 7]
```diff
@@ -10,7 +10,51 @@ guiding-principles:
...
+  en: 'File an issue'
+issues_description:
```
Comment (rowan-m) [inline site/_data/i18n/docs/privacy-sandbox.yml:25]: Unused.

File: site/_data/i18n/docs/privacy-sandbox.yml [hunk 8]
```diff
@@ -10,7 +10,51 @@ guiding-principles:
...
+  en: 'Read the proposal, raise questions, and follow discussion.'
+timeline:
+  en: 'Timeline'
+timeline-descrition:
+  en: 'Discover when we expect new APIs and technologies to be ready for adoption.'
+blog:
+  en: 'Blog'
+subscribe:
```
Comment (rowan-m) [inline site/_data/i18n/docs/privacy-sandbox.yml:33]: Unused.

File: site/_data/i18n/docs/privacy-sandbox.yml [hunk 9]
```diff
@@ -10,7 +10,51 @@ guiding-principles:
...
+  en: 'Get started'
+discuss:
+  en: 'Discussion and updates'
+issues:
+  en: 'File an issue'
+issues_description:
+  en: 'Read the proposal, raise questions, and follow discussion.'
+timeline:
+  en: 'Timeline'
+timeline-descrition:
```
Comment (rowan-m) [inline site/_data/i18n/docs/privacy-sandbox.yml:29]: Typo in `descrition` but also maybe unused?

File: site/_data/i18n/docs/privacy-sandbox.yml [hunk 10]
```diff
@@ -10,7 +10,51 @@ guiding-principles:
...
+  en: 'Blog'
+subscribe:
+  en: 'Subscribe to RSS'
+attribution-reporting-overview:
+  en: 'Overview'
+attribution-reporting-events:
+  en: 'Event-level reports'
+attribution-reporting-events-api:
+  en: 'Event-level reports API guide'
+attribution-reporting-faqs:
```
Comment (rowan-m) [inline site/_data/i18n/docs/privacy-sandbox.yml:41]: Unused and also doesn't seem quite right either, FAQ paired with a description about data clearing.

**Review by rowan-m — CHANGES_REQUESTED** [review]
Looking good, just some minor clean-up.

**Review by rowan-m — APPROVED** [review]
LGTM

**Pull request merged**